src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/SymbolCompletionProviderTests.vb

```diff
@@ -5713,6 +5713,38 @@ Class SomeClass
     Inherits $$
 
 End
+"
+
+            Await VerifyItemExistsAsync(markup, "somestruct")
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestInherits_QualifiedNameAsGenericParameter() As Task
+            Const markup = "
+Namespace MyNameSpace
+    Structure somestruct
+    End Structure
+End Namespace
+
+Class SomeClass
+    Inherits List(Of MyNameSpace.$$)
```

Comment:
can we have tests where there is `list(of $$` or `List(of mynamespace.$$.Foo`?

Reply:
These cases are not restricted by inheritance rules. It is perfectly fine to have a class, that inherits a list of integers, while deriving from and integer is invalid

Reply:
yes.  i understand.  i was just saying that we should have tests for this :)

Reply:
the tests i asked for are still needed.